### PR TITLE
Adding VIA support for CannonKeys/Practice60

### PR DESCRIPTION
**Pull request by morbetter**
## Description

Created JSON for CannonKeys/Practice60

## Types of Changes

- [X] Adding a keyboard to VIA
- [ ] Changing a keyboard already in VIA
- [ ] Bug fix
- [ ] Documentation

## Checklist

- [X] I have tested this keyboard definition using VIA's "Design" tab.
- [X] I have tested this keyboard definition with firmware on a device.
- [X] The QMK firmware can be built using QMK master repo.

### CODE REVIEW

**Comment by unicorn855** [comment]
Idk why this has been sitting here for almost four months now, but you may want to review the (updated) checklist again @morbetter (does something else needs reworking @olivia ?

**Comment by morbetter** [comment]
@unicorn855 It's probably waiting for the [QMK pull request](https://github.com/qmk/qmk_firmware/pull/8169) to be approved and merged.

**Comment by unicorn855** [comment]
Probably :-) I just wondered because it has been some time since the qmk PR 🤷‍♀️

**Comment by olivia** [comment]
@unicorn855 , @morbetter is correct. We require the QMK side to land first, or else we wouldn't be able to generate the firmware and validate it.

**Comment by unicorn855** [comment]
> @unicorn855 , @morbetter is correct. We require the QMK side to land first, or else we wouldn't be able to generate the firmware and validate it.

Yes, I understand that :) I meant I  think there are suggestions still open in the original qmk PR :) but thank you for that reply @olivia

**Comment by morbetter** [comment]
@unicorn855 , the PR was approved and merged, FYI.

**Comment by unicorn855** [comment]
> @unicorn855 , the PR was approved and merged, FYI.

Ah okay I see :-)

**Comment by unicorn855** [comment]
> > @unicorn855 , the PR was approved and merged, FYI.
> 
> Ah okay I see :-)

I guess @olivia is busy with things :-)

File: src/other/practice60/practice60.json [hunk 1]
```diff
@@ -0,0 +1,149 @@
+{
+  "name": "Practice60",
+  "vendorId": "0xCA04",
+  "productId": "0x5060",
```
Comment (jkutianski) [inline src/other/practice60/practice60.json:4]: The productId on the firmware is 0x6060

**Comment by jkutianski** [comment]
The PID is incorrect! On the JSON is 5060 but the binfrom VIA is 6060! 
https://github.com/the-via/keyboards/pull/49/commits/12b7f1ab44a57cbf19d435f0b827aa91f728a779#r464000330

File: src/other/practice60/practice60.json (hunk 1 above)
Comment (morbetter) [reply src/other/practice60/practice60.json]: Updated

**Comment by morbetter** [comment]
Bump?

**Pull request merged**